cukinia

```diff
@@ -822,6 +823,32 @@ _cukinia_gpio_sysfs()
 	return 0
 }
 
+# _cukinia_knoerror: Check the kernel has booted without important errors
+# arg1: the message priority
+_cukinia_knoerror()
+{
+	local priority="$1"
+	local ret
+	local use_journalctl
+
+	which journalctl >> /dev/null
+	use_journalctl=$?
```

Comment:
avoid clever tricks like this, keep the script obvious

 if something; then use_journalctl=1 

the fact it will use it if its value is 0 is also counter intuitive.